tests/p2p/test_split_brain.py

```diff
@@ -268,12 +263,10 @@ def test_split_brain_only_blocks_same_height(self):
         self.assertEqual(len(manager2.tx_storage.get_best_block_tips()), 1)
         self.assertCountEqual(manager2.tx_storage.get_best_block_tips(), {new_block.hash})
 
-    def test_split_brain_only_blocks_bigger_score(self):
+    def test_split_brain_only_blocks_bigger_score(self) -> None:
         manager1 = self.create_peer(self.network, unlock_wallet=True)
-        manager1.avg_time_between_blocks = 3
```

Comment:
Why are you changing this?

Reply:
After this test was typed, mypy complained that `HathorManager` does not have a `avg_time_between_blocks` property. Therefore, this was useless (probably an artifact from some older code) and removing it doesn't break the test.

Reply:
This line was commented instead. So, we are going to fix it later.